R/utils.R

```diff
@@ -44,10 +52,10 @@ clean_regional_data <- function(cases) {
 }
 
 #' Regional EpiNow with settings
-regional_epinow_with_settings <- function(reported_cases, generation_time, delays, 
```

Comment:
some of the defaults from update_regional could be moved into here (like the standard delays used etc.) or this function could be moved into update_regional. Not sure if there needs to be this additional funciton layer now.

Reply:
possibly not - it might make sense to merge them together. Initially I was just trying to pull all the duplicate code from each location into a common method. I think merging these two together would be sensible as a next step but I might do it as a follow on task (this is one of those awkward changes that restructures things so whilst simple still means a big step from current master so the less time it's in progress the better)